crates/holochain/tests/inline_zome_sketch.rs

```diff
@@ -34,145 +48,109 @@ fn simple_crud_zome() -> InlineZome {
+
+    // Install DNA and install and activate apps in conductor
+    let ids = conductor
+        .setup_app_for_all_agents_with_no_membrane_proof(
+            "app",
+            &[alice.clone(), bobbo.clone()],
+            &[dna_file],
+        )
+        .await;
 
-    let conductor = Conductor::builder().test(&envs).await?;
+    let ((alice,), (bobbo,)) = destructure_test_cells!(ids);
 
-    // Install the DNA
+    // Call the "create" zome fn on Alice's app
+    let hash: HeaderHash = alice.call("zome1", "create_unit", ()).await;
 
-    conductor.install_dna(dna_file).await?;
+    // Wait long enough for Bob to receive gossip (TODO: make deterministic)
+    tokio::time::delay_for(std::time::Duration::from_millis(500)).await;
 
-    // Install and activate one app for Alice and another for Bob
-    // TODO: develop tools to app installation much less verbose
+    // Verify that bobbo can run "read" on his cell and get alice's Header
+    let element: MaybeElement = bobbo.call("zome1", "read", hash).await;
+    let element = element
+        .0
+        .expect("Element was None: bobbo couldn't `get` it");
 
-    conductor
-        .clone()
-        .install_app(
-            "app:alice".to_string(),
-            vec![(
-                InstalledCell::new(alice_cell_id.clone(), "dna".into()),
-                None,
-            )],
-        )
-        .await?;
-    conductor
-        .clone()
-        .install_app(
-            "app:bobbo".to_string(),
-            vec![(
-                InstalledCell::new(bobbo_cell_id.clone(), "dna".into()),
-                None,
-            )],
-        )
-        .await?;
-    conductor.activate_app("app:alice".to_string()).await?;
-    conductor.activate_app("app:bobbo".to_string()).await?;
-    conductor.clone().setup_cells().await?;
+    // Assert that the Element bobbo sees matches what alice committed
+    assert_eq!(element.header().author(), alice.agent_pubkey());
+    assert_eq!(
+        *element.entry(),
+        ElementEntry::Present(Entry::app(().try_into().unwrap()).unwrap())
+    );
 
-    // Call the "create" zome fn on Alice's app
-    // TODO: develop tools to make zome calls much less verbose
-
-    let hash: HeaderHash = {
-        let response = conductor
-            .call_zome(ZomeCall {
-                cell_id: alice_cell_id.clone(),
-                zome_name: zome.zome_name().clone(),
-                fn_name: "create".into(),
-                payload: ExternInput::new(().try_into().unwrap()),
-                cap: None,
-                provenance: alice.clone(),
-            })
-            .await??;
-        unwrap_to!(response => ZomeCallResponse::Ok)
-            .clone()
-            .into_inner()
-            .try_into()?
-    };
-
-    // Wait long enough for Bob to receive gossip
+    Ok(())
+}
+
+#[tokio::test(threaded_scheduler)]
+#[cfg(feature = "test_utils")]
+async fn inline_zome_3_agents_2_dnas() -> anyhow::Result<()> {
```

Comment:
The diff in this file would have been much more dramatic (and helpful) if I hadn't added a second test. Note the huge boilerplate removal of the first test, though.